README.md

```diff
@@ -18,4 +18,11 @@ Let's pick the terms for 2017!! The rules are simple, PR your name and your top
 * micro containers
 * smart data
 
+### [Waldo's](https://github.com/gwaldo) picks
```

Comment:
Sir, I would like to donate to your political campaign. Do you take tear stained bills?